resources/lang/en/general.php

```diff
@@ -264,7 +264,7 @@
     'accept_eula'           => 'Acceptance Agreement',
     'supplier'              => 'Supplier',
     'suppliers'  			=> 'Suppliers',
-    'sure_to_delete'    => 'Are you sure you wish to delete',
+    'sure_to_delete'    => 'Are you sure you wish to delete :item?',
```

Comment:
so this was fixed in my local? so i slapped a ? at the end and pushed the new commit

Reply:
We don't actually want a ? there, since we tack on (via PHP and JS) the specific item name. Grep through the code for the places we use `sure_to_delete` as a translated string and you'll see what I mean. It would end up with "Are you sure you wish to delete? Asset #1234o8568?"

Reply:
oh this is why it loked the way it did when i pulled it up. yea easy fix.